ci/test/e2e.go

```diff
@@ -118,16 +141,29 @@ func (r *runner) setup() error {
 	if err := r.compose("build"); err != nil {
 		return errors.Wrap(err, "building app images failed!")
 	}
+
+	return nil
+}
+
+func (r *runner) setupBasicEnv() error {
 	log.Info("starting app containers")
-	if err := r.compose("up", "-d", "myst-provider", "myst-consumer", "ipify"); err != nil {
+	if err := r.compose("up", "-d", "myst-provider", "myst-consumer"); err != nil {
 		return errors.Wrap(err, "starting app containers failed!")
 	}
-
 	return nil
 }
 
 func (r *runner) test() error {
-	log.Info("running tests")
+	log.Info("running tests for env: ", r.testEnv)
+
+	var consumerServices string
+
+	if r.testEnv == "basic" {
+		consumerServices = "openvpn,noop,wireguard"
```

Comment:
pass these as a constructor arg to the runner itself? no need to check it here then.